SplunkAppForWazuh/bin/wazuh_api.py

```diff
@@ -35,6 +35,7 @@ def __init__(self):
             self.wztoken = wazuhtoken()
             self.session = requests.Session()
             self.session.trust_env = False
+            self.timeout = int(self.config['timeout'])

```

Comment:
**nitpick**: `self.config` is not defined before using it.

Reply:
Solved in https://github.com/wazuh/wazuh-splunk/pull/1199